src/NewTwitchApi/Resources/StreamsApi.php

```diff
@@ -11,15 +11,15 @@ class StreamsApi extends AbstractResource
 {
     /**
      * @throws GuzzleException
-     */
+   */
```

Comment:
Why? This looks wrong. Please fix this everywhere. I see it in many places in this PR.

Reply:
As requested - you can see the updates below.

The code, for the most part, has been easy to follow, however, I'm noticing that maybe its missing the POST, PUT and DELETE functionality? This may be a limitation of my understanding of the library, but that's why I avoided completing the Endpoints that use protocols other than GET. I have found the calls easy to use in my projects. I'd like to find a more elegant way to pass/store the Bearer token and use it across multiple calls instead of having to pass it every time, but this is more a personal preference, but also when coding I noticed it problematic to decide where to place the token given the number of required, OR, or optional parameters. 

Most of the issues I run into are less with the library and more with Twitch. I find their documentation lacking in details on newer endpoints and not providing endpoints required for developers. They've recently been doing a better job of allowing developers access to Twitch Staff through things like their forums and also the new Discord they launched recently, which I got quite a bit of support from to ask clarifying questions, but this is more recent.

One thing I've had a really hard time dealing with is exceptions. I understand and can see that errors throw Guzzle Exceptions, but I'm at a complete loss for how to catch them. I have not put in the effort I should, to figure out how to do this, but I would say it would be unclear and not-achievable for beginning developers who rely on things like libraries to get them by, and sometimes even help them learn more.

Thanks for asking for my feedback, I hope to continue to contribute and help in any way I can. Feel free to reach out anytime.

Brandin.

Thank you,
Brandin.